src/api-server-selector.js

```diff
@@ -53,6 +53,11 @@ export class ApiServerSelector extends EventsTargetMixin(AmfHelperMixin(LitEleme
        * AMF model to be rendered
        */
       amf: { type: Object },
+      /**
+       * If this property is set, the dropdown and the url input are
+       * rendered in the same line
+       */
+      inline: { type: Boolean },
```

Comment:
Technically this is correct. Alternative would be to reflect this value to an attribute (`reflect: true` config property). Then you can use `:host([inline]) {}` css selector to style internal DOM directly from attributes. 
Both ways are valid so just mentioning the alternative.

Reply:
Interesting, wasn't aware of this. Will keep in mind for the future